app/models/manageiq/providers/autosde/storage_manager/storage_service.rb

```diff
@@ -17,25 +18,25 @@ def self.raw_create_storage_service(ext_management_system, options = {})
 
     task_id = ext_management_system.autosde_client.ServiceApi.services_post(service_to_create).task_id
     options = {
-      :target_class   => :storage_services,
+      :target_class   => nil,
       :target_id      => nil,
       :ems_id         => ext_management_system.id,
       :native_task_id => task_id,
       :interval       => 10.seconds,
-      :target_option  => "new"
+      :target_option  => "ems"
     }
     ext_management_system.class::EmsRefreshWorkflow.create_job(options).tap(&:signal_start)
   end
 
   def raw_delete_storage_service
     task_id = ext_management_system.autosde_client.ServiceApi.services_pk_delete(ems_ref).task_id
     options = {
-      :target_id      => id,
-      :target_class   => self.class.name,
+      :target_id      => nil,
+      :target_class   => nil,
       :ems_id         => ext_management_system.id,
       :native_task_id => task_id,
       :interval       => 20.seconds,
-      :target_option  => "existing"
+      :target_option  => "ems"
```

Comment:
Same here, this looks unrelated to the rest of the changes and delete should be a targeted refresh.

Reply:
I've opened #220 for this particular change and will remove it from this PR